### ISSUE TITLE
ParseError: no element found: line 45, column 0

### ISSUE THREAD

**Issue by maplewizard** [RaRe-Technologies/gensim#1496]
Hi, I downloaded data from https://dumps.wikimedia.org/enwiki/20170701/enwiki-20170701-pages-articles-multistream.xml.bz2
and I run code as:
```
from gensim.corpora import WikiCorpus
inp = "enwiki-20170701-pages-articles-multistream.xml.bz2"
wiki = WikiCorpus(inp, lemmatize=False, dictionary={})
a = wiki.get_texts()
a.next()
```
and I got 
```
Process InputQueue-24:
Traceback (most recent call last):
  File "/home/tkzif/ProgramFiles/anaconda2/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/home/tkzif/ProgramFiles/anaconda2/lib/python2.7/site-packages/gensim/utils.py", line 843, in run
    wrapped_chunk = [list(chunk)]
  File "/home/tkzif/ProgramFiles/anaconda2/lib/python2.7/site-packages/gensim/corpora/wikicorpus.py", line 310, in <genexpr>
    ((text, self.lemmatize, title, pageid)
  File "/home/tkzif/ProgramFiles/anaconda2/lib/python2.7/site-packages/gensim/corpora/wikicorpus.py", line 215, in extract_pages
    for elem in elems:
  File "/home/tkzif/ProgramFiles/anaconda2/lib/python2.7/site-packages/gensim/corpora/wikicorpus.py", line 200, in <genexpr>
    elems = (elem for _, elem in iterparse(f, events=("end",)))
  File "<string>", line 107, in next
ParseError: no element found: line 45, column 0
```
Is there any thing wrong with my method?

**Comment by xurannlpr** [comment]
i also meet this error, have you already fixed it?

**Comment by maplewizard** [comment]
@xurannlpr yes， I do fixed the error. It is due to my error download package. Please download the package exactly as the document in gensim. It may help. Thanks.

**Comment by polm** [comment]
Looks like the issue is that "multistream" isn't supported. Seems this has confused other people too...

https://groups.google.com/forum/#!msg/gensim/vA97TdOkljk/ty1YPuZWAQAJ

**Comment by menshikh-iv** [comment]
Resolved in #1515
